Node/src/TeamsChatConnector.ts

```diff
@@ -110,6 +112,17 @@ export class TeamsChatConnector extends builder.ChatConnector {
   }
 
   /**
+  * This method is to resolve regex or LUIS Bot being affected by at mentions
+  * Set true would have incoming messages stripped off source Bot at mentions
+  * The default would be false
+  * Developer could swithc on/off anytime
+  * @param {boolean} flag - switch for filter Bot at mentions.
+  */
+  public setUsingIntentText(flag: boolean) {
```

Comment:
setUsingIntentText isn't clear. I'd name this "stripBotAtMentions(boolean)"

Or maybe it'd be better to follow the BotBuilder pattern, which is to take a settings object in the constructor. Have an ITeamsChatConnectorSettings that extends IChatConnectorSettings with a stripBotAtMentions property.